### PR TITLE
Even More zNPC Void Functions

### PR DESCRIPTION
**Pull request by TotallyNotMichael-GH**

### CODE REVIEW

File: src/SB/Game/zAssetTypes.cpp [hunk 1]
```diff
@@ -63,3 +63,7 @@ void* JSP_Read(void* param_1, U32 param_2, void* indata, U32 insize, U32* outsiz
     jsp_shadow_hack(retjsp);
     return retjsp;
 }
+
+void Anim_Unload(void*, unsigned int)
```
Comment (mattbruv) [inline src/SB/Game/zAssetTypes.cpp]: should be `U32` instead of `unsigned int`.

File: src/SB/Game/zEntDestructObj.cpp [hunk 2]
```diff
@@ -1,3 +1,7 @@
 #include "zEntDestructObj.h"
 
 #include <types.h>
+
+void zEntDestructObj_Move(zEntDestructObj*, xScene*, float, xEntFrame*)
```
Comment (mattbruv) [inline src/SB/Game/zEntDestructObj.cpp]: `F32` instead of `float`

File: src/SB/Game/zFX.cpp [hunk 3]
```diff
@@ -27,6 +27,10 @@ extern F32 lbl_803CD968; // 0.15f
 extern F32 lbl_803CD96C; // 12.0f
 extern F32 lbl_803CD970; // 2.0f
 
+void xDrawSphere2(const xVec3*, float, unsigned int)
```
Comment (mattbruv) [inline src/SB/Game/zFX.cpp]: `F32, U32`, etc.

File: src/SB/Game/zLightning.cpp [hunk 4]
```diff
@@ -92,14 +92,33 @@ void xDebugAddFlagTweak(char* name, U32* property, U32 param_3, tweak_callback*
...
+                    void* param_6, U32 param_7)
 {
     // Redacted. :}
 }
+
+void xDebugAddTweak(const char*, short*, short, short, const tweak_callback*, void*, unsigned int)
+{
+}
+
+void xDebugAddSelectTweak(const char*, unsigned int*, const char**, const unsigned int*,
```
Comment (mattbruv) [inline src/SB/Game/zLightning.cpp]: `char` should be `S8`, or a signed 8 bit integer.

**Review by mattbruv — CHANGES_REQUESTED** [review]
Basically do these changes for all of the primitive types

File: src/SB/Game/zEntSimpleObj.cpp [hunk 5]
```diff
@@ -1,3 +1,7 @@
 #include "zEntSimpleObj.h"
 
 #include <types.h>
+
+void zEntSimpleObj_Move(xEnt*, xScene*, float, xEntFrame*)
```
Comment (mattbruv) [inline src/SB/Game/zEntSimpleObj.cpp]: F32 here

File: src/SB/Game/zLightEffect.cpp [hunk 6]
```diff
@@ -139,6 +139,54 @@ void zLightEffectRandomColFast()
...
+void zLightEffectRandomColSlow(_zLight*, float)
+{
+}
+
+void zLightEffectHalfDimFast(_zLight*, float)
+{
+}
+
+void zLightEffectHalfDim(_zLight*, float)
+{
```
Comment (mattbruv) [inline src/SB/Game/zLightEffect.cpp]: All of these floats need to be F32

File: src/SB/Game/zNPCMessenger.cpp [hunk 7]
```diff
@@ -1,3 +1,15 @@
 #include "zNPCMessenger.h"
 
 #include <types.h>
+
+void zNPCCommon::Stun(float)
```
Comment (mattbruv) [inline src/SB/Game/zNPCMessenger.cpp]: F32

File: src/SB/Game/zThrown.cpp [hunk 8]
```diff
@@ -135,6 +134,6 @@ void checkAgainstButtons(xEnt* ent)
 
 #endif
 
-void xDrawSphere(xSphere* s, U32 unk)
+void xDrawSphere(const xSphere*, unsigned int)
```
Comment (mattbruv) [inline src/SB/Game/zThrown.cpp]: U32

**Review by mattbruv — CHANGES_REQUESTED** [review]

**Review by TotallyNotMichael-GH — COMMENTED** [review]
changed made as asked..

**Pull request merged**